packages/devtools/lib/src/timeline/cpu_flame_chart.dart

```diff
@@ -172,15 +103,19 @@ class CpuFlameChart extends CoreElement {
     }
   }
 
-  Future<void> show() async {
+  void show() async {
     attribute('hidden', false);
 
     if (canvasNeedsRebuild) {
       canvasNeedsRebuild = false;
-      await update();
+      update();
     }
   }
 
+  void hide() {
+    attribute('hidden', true);
```

Comment:
there are a ton of calls to 
attribute('hidden', value);
Maybe make a default implementation of show and hide in a base class that has that implementation?

Reply:
convention also appears to be that there are show and hide methods everywhere so might as well make that part of an official recommended api.

Reply:
looks like the CoreElement class already has this `hidden(bool value)`. I'll go ahead and use that.